### PR TITLE
Update README.md regarding abseil dependencies and add fix for owners…

### PR DESCRIPTION
**Pull request by jplevyak**
…hip.

Signed-off-by: John Plevyak <jplevyak@gmail.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: api/wasm/cpp/README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ docker run -v $PWD:/work -v $PWD/../envoy/api/wasm/cpp:/work/sdk -w /work  wasms
 
 ### Using abseil form the image
 
-Abseil (optionally) is built in /root/abseil and can be used by customizing the Makefile e.g.:
+Abseil (optionally) is built in /root/abseil and can be used.  Note that the abseil containers (e.g. flat\_hash\_set) exercise many syscalls which are not supported. Consequantally individual files should be pulled in which are relatively self contained (e.g. strings). Example customized Makefile:
```
Comment (PiotrSikora) [inline api/wasm/cpp/README.md]: Nit: `absl::flat_hash_set`

File: api/wasm/cpp/README.md [hunk 2]
```diff
@@ -127,15 +127,22 @@ all: plugin.wasm
...
 ```
-DOCKER_SDK=/work/sdk
+DOCKER_SDK=/sdk
 
 all: myproject.wasm
-  chown ID.ID $^
+  chown ${uid}.${gid} $^
 
-include ${DOCKER_SDK}/Makefile.base_lite
+include /sdk/Makefile.base_lite
```
Comment (PiotrSikora) [inline api/wasm/cpp/README.md]: Why this was changed from `${DOCKER_SDK}` to a hardcoded value?

File: api/wasm/cpp/README.md (hunk 1 above)
Comment (jplevyak) [reply api/wasm/cpp/README.md]: Done.

File: api/wasm/cpp/README.md (hunk 2 above)
Comment (jplevyak) [reply api/wasm/cpp/README.md]: Done

**Pull request merged**